### ISSUE TITLE
No longer support armv7?

### ISSUE THREAD

**Issue by wolverinevn** [danielwelch/hassio-zigbee2mqtt#382]
**zigbee2mqtt add-on version (if edge, please report commit hash):**
1.14.4

**Operating environment (HassOS, Virtual Machine, Device/platform running Hass.io):**
Home Assistant 0.114.0
Supervisor 242
Docker version 19.03.12, build 48a6621
Raspbian GNU/Linux 10 (buster) armv7l GNU/Linux

**Description of problem:**
This add-on is not available on your system.

**Your entire configuration from the frontend (with sensitive fields redacted)**:
```json
// replace this line with your pasted configuration
```

**Your logs from Hass.io**

```
20-09-11 02:25:35 INFO (MainThread) [supervisor.store.git] Clone add-on https://github.com/danielwelch/hassio-zigbee2mqtt repository
20-09-11 02:25:39 INFO (MainThread) [supervisor.store] Load add-ons from store: 80 all - 2 new - 0 remove
20-09-11 02:25:46 ERROR (MainThread) [supervisor.addons] Add-on 7ad98f9c_zigbee2mqtt not supported on that platform
20-09-11 02:25:52 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/Poeschl/Hassio-Addons repository
20-09-11 02:25:52 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/danielwelch/hassio-zigbee2mqtt repository
```

**Comment by ciotlosm** [comment]
Was this working in the past?

**Comment by wolverinevn** [comment]
Yes. Installed and used it flawlessly until a week ago. It just disappeared from add-on store (reposity is already added).
But I can find and install it on same Pi, same Raspbian, same Home Assistant with 64bit kernel (so aarch64).

**Comment by ciotlosm** [comment]
The image seems to be there: https://hub.docker.com/r/dwelch2101/zigbee2mqtt-aarch64. Maybe some local caching. Try removing and adding the repository itself.

**Comment by ciotlosm** [comment]
This might be the same as #366 so I linked it there. If you are no longer experiencing the issue it's probably related.

**Comment by wolverinevn** [comment]
Thanks but no image for armv7. Is it removed or never been there? But we all used it on Raspbian before.

**Comment by ciotlosm** [comment]
Maybe it was using armhf before and something changed in HA that no longer knows to use armhf image?
According to https://superuser.com/questions/1009540/difference-between-arm64-armel-and-armhf armhf should work.

Maybe you have a problem closer to: https://community.home-assistant.io/t/installing-hassio-with-docker/101118

**Comment by danielwelch** [comment]
See this relevant PR here: https://github.com/danielwelch/hassio-zigbee2mqtt/pull/124

My understanding has been since then that `armv7` would "fall back" to `armhf` if not specified as a supported architecture, and that seems to have been working for quite some time. Will have to check if something changed within HA.

**Comment by wolverinevn** [comment]
> See this relevant PR here: #124
> 
> My understanding has been since then that `armv7` would "fall back" to `armhf` if not specified as a supported architecture, and that seems to have been working for quite some time. Will have to check if something changed within HA.
Should be related to some recent Supervisor updates? This is my first though but some other add-ons work well and I do not have much time to investigate. Move zigbee2mqtt to docker container anyway.

**Comment by danielwelch** [comment]
This is tentatively fixed by #388. I'll close for now, but please feel free to re-open if there are continued issues with armv7 architecture
